aptos-move/framework/src/built_package.rs

```diff
@@ -166,6 +177,25 @@ impl BuiltPackage {
     pub fn build(package_path: PathBuf, options: BuildOptions) -> anyhow::Result<Self> {
         let bytecode_version = options.bytecode_version;
         let compiler_version = options.compiler_version;
+        let language_version = options.language_version;
+        let effective_compiler_version = compiler_version.unwrap_or_default();
```

Comment:
Need those `compiler_version` unwrapped instances a bit later in the code, hence can't overload the name. II moved this code into a helper function to avoid the issue.